src/ogc-api/endpoint.ts

```diff
@@ -130,6 +147,29 @@ ${e.message}`);
       .then(parseCollections('feature'));
   }
 
+  /**
+   * A Promise which resolves to an array of vector tile collection identifiers as strings.
+   */
+  get vectorTileCollections(): Promise<string[]> {
```

Comment:
I'm wondering if there would be a way to get a list of collections that support vector tiles without having to query each of them; `getCollectionInfo` will do a separate HTTP query for each and it is currently not cached, there might be a better way to get that list?